### DISCUSSION TITLE
Main Menu with Access Point Mode Only

### DISCUSSION THREAD

**Discussion by kebaranas** [arendst/Tasmota#18286] [Q&A] [answered]
Is there a way to skip the WiFiManager upon connecting to the Tasmota device's access point. Basically, you don't have to connect the Tasmota device to an access point.

**Comment by Jason2866** [comment]
Tasmota does not work without a connection to a wifi network since it has to reach an NTP server.

**Comment by pkkrusty** [comment, marked as answer]
See #13362 for a workaround. Requires self-compiling after adjusting some code, but generally works. Especially if you use a time source in conjunction (like DS3231 or GPS)

**Reply by kebaranas** [reply]
It worked with Tasmota 12.4.0. Thank you!
